include/klee/Internal/Module/KInstruction.h

```diff
@@ -44,7 +44,8 @@ namespace klee {
 
   public:
     virtual ~KInstruction();
-    void printFileLine(llvm::raw_ostream &);
+    void printFileLine(llvm::raw_ostream &) const;
+    std::string printFileLine() const;
```

Comment:
`printFileLine()` isn't a very appropriate name given that it doesn't print to a stream. Wouldn't `getFileLine()` be better?